qdrant-landing/content/use-cases/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Vector Database Use Cases
-section_title: Apps and Ideas Qdrant made possible
+title: Vector Database Use Cases | Qdrant
```

Comment:
We add the word "Qdrant" to the title automatically, so this will create duplication. 
> Vector Database Use Cases | Qdrant - Qdrant